cmd/xcode.go

```diff
@@ -108,6 +114,67 @@ func scanXcodeProject(_ *cobra.Command, _ []string) error {
 		xcodeCmd.SDK = paramXcodebuildSDK
 	}
 
+	if paramXcodeDestination != "" {
+		xcodeCmd.DESTINATION = paramXcodeDestination
+	} else {
+		var project xcodeproj.XcodeProj
+		var scheme xcscheme.Scheme
+
+		if xcodeproj.IsXcodeProj(xcodeCmd.ProjectFilePath) {
+			proj, err := xcodeproj.Open(xcodeCmd.ProjectFilePath)
+			if err != nil {
+				return fmt.Errorf("Failed to open project (%s), error: %s", xcodeCmd.ProjectFilePath, err)
+			}
+
+			projectScheme, _, err := proj.Scheme(xcodeCmd.Scheme)
```

Comment:
The configuration used for xcodebuild calls can be different than the scheme's default configuration, it is better to explicitly specify the configuration for xcodebuild calls.